Pod/Classes/Pipes.swift

```diff
@@ -52,9 +52,8 @@ Try piping the optional input into the function on the right.
 
 Result is nil if lhs is nil. Otherwise, same as `|>`
 */
-public func |?> <A, B>(lhs: A?, rhs: A -> B) -> B? {
-    return lhs
-        |> map { rhs($0) }
+public func |?> <A, B>(lhs: A?, @noescape rhs: A -> B) -> B? {
+  return lhs.map(rhs)
```

Comment:
On the other hand, the project did not compile with XCode 6.4 before, because the same `.map` calls are also used in `Currying.swift`. So we would have to rewrite these, too.